### PR TITLE
Minor edits to UG

### PR DESCRIPTION
**Pull request by tsy24**
Edit link in Section 2
Move storage to new section

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -405,7 +405,7 @@ Examples:
 
 :bulb: **Tip:**  
 
-* You can remove a remark by leaving the remark input empty! Eg. `remark 1 re/`
+* You can remove a remark by leaving the remark input empty! eg. `remark 1 re/`
```
Comment (xiongjya) [inline docs/UserGuide.md:408]: How about e.g. instead of eg.?

**Review by xiongjya — CHANGES_REQUESTED** [review]
Under the `viewSchedule` section, there are also `E.g.` instead of `e.g.`, so do remember to change from capital letter to small. Otherwise, it looks good!

**Review by nicole-luo-exe — APPROVED** [review]
LGTM!

**Review by nicole-luo-exe — APPROVED** [review]
Looks good

**Review by xiongjya — APPROVED** [review]
LGTM!

**Pull request merged**